tokio-trace-serde/README.md

```diff
@@ -0,0 +1,95 @@
+# tokio-trace-serde
+
+An adapter for serializing `tokio-trace` types using `serde`.
+
+[Documentation](https://docs.rs/tokio-trace-serde/0.1.0/tokio_trace_serde/index.html)
+
+## Overview
+
+`tokio-trace-serde` enables serializing `tokio-trace` types using
+`serde`. `tokio-trace` is a framework for instrumenting Rust programs
+to collect structured, event-based diagnostic information.
+
+Traditional logging is based on human-readable text messages.
+`tokio-trace` gives us machine-readable structured diagnostic
+information. This lets us interact with diagnostic data
+programmatically. With `tokio-trace-serde`, you can implement a
+`Subscriber` to serialize your `tokio-trace` types and make use of the
+existing ecosystem of `serde` serializers to talk with distributed
+tracing systems.
+
+Serializing diagnostic information allows us to do more with our logged
+values. For instance, when working with logging data in JSON gives us
+pretty-print when we're debugging in development and you can emit JSON
+and tracing data to monitor your services in production.
+
+The `tokio-trace` crate provides the APIs necessary for instrumenting
+libraries and applications to emit trace data.
+
+## Usage
+
+First, add this to your `Cargo.toml`:
+
+```toml
+[dependencies]
+tokio-trace = "0.1"
+tokio-trace-serde = "0.1"
+```
+
+Next, add this to your crate:
+
+```toml
+#[macro_use]
+extern crate tokio_trace;
+extern crate tokio_trace_serde;
+
+use tokio_trace_serde::AsSerde;
+```
+
+Please read the [`tokio-trace` documentation](https://docs.rs/tokio-trace/0.1.0/tokio_trace/index.html)
+for more information on how to create trace data.
+
+This crate provides the `as_serde` function, via the `AsSerde` trait,
```

Comment:
When the crate is published, it would be nice if this linked to the `docs.rs` documentation for `as_serde`...not a change we can make now, though.

Reply:
I'll remember to make that change!